go/vt/throttler/throttler.go

```diff
@@ -224,6 +225,28 @@ func (t *Throttler) Throttle(threadID int) time.Duration {
 	return t.threadThrottlers[threadID].throttle(t.nowFunc())
 }
 
+// LastMaxLagNotIgnoredForTabletType returns the max of all the last replication lag values seen across all tablets of
```

Comment:
This should be edited so that the comment matches the function name.

Reply:
Should be fixed now.